launcher/image/test/test_hardened_unstable_cloudbuild.yaml

```diff
@@ -0,0 +1,41 @@
+substitutions:
+  # Expects hardened image (not debug) and should have startup-script service
+  # disabled. google-startup-scripts.service is only enabled with multi-user.target.
+  '_IMAGE_NAME': ''
+  '_IMAGE_PROJECT': ''
+  # Add user-data in the metadata to test if it is disabled.
+  '_METADATA_FILE': 'startup-script=data/echo_startupscript.sh,user-data=data/cloud-init-config.yaml'
```

Comment:
Should we have startupscript enabled here? because seem like it is only testing os policy here